CONTRIBUTING.md

```diff
@@ -1,10 +1,22 @@
 # Contributing
 
+## Introduction
+
+This book is a catalogue of Rust programming techniques, (anti-)patterns, and idioms and other explanations.
+It is a compilation of collective (sometimes implicit) knowledge or experience that was created with collaborative effort.
+
+These contained information are __no rules__, they should be understood as guidelines.
```

Comment:
```suggestion
The patterns described here are __not rules__, but should be taken as guidelines to when writing code in Rust.
```

Reply:
```suggestion
The patterns described here are __not rules__, but should be taken as guidelines when writing code in Rust.
```

Reply:
```suggestion
The patterns described here are __not rules__, but should be taken as guidelines for writing idiomatic code in Rust.
```

?

Reply:
That sounds much better to me, didn't even notice the extra `to` there, thanks pickfire!

Reply:
So pickfire's version it should be then, I assume?